index.html

```diff
@@ -951,10 +965,6 @@ <h3><dfn data-lt="implicitly release the pointer capture|implicitly releasing po
                After running <a>process pending pointer capture</a> steps,
                if the pointer supports hover, user agent MUST also send corresponding boundary events necessary
                to reflect the current position of the pointer with no capture.</p>
-            <p>If the user agent supports firing the <code>click</code> or <code>auxclick</code> event,
-               and if in an implicit release scenario both that (<code>click</code> or <code>auxclick</code>)
-               event and a {{GlobalEventHandlers/lostpointercapture}} event are fired,
-               the <code>click</code> or <code>auxclick</code> event SHOULD be fired before {{GlobalEventHandlers/lostpointercapture}}.</p>
             <p>When the <a>pointer capture target override</a> is no longer [=connected=] [[DOM]],
```

Comment:
Just spotted that the "target override no longer connected" could apply to click-like events too!  I will update the PR shortly.